src/types/User/index.ts

```diff
@@ -0,0 +1,22 @@
+export interface UserEntity {
+    id: string;
+    firstname: string;
+    lastname: string;
+    email: string;
+    createdAt: Date;
+    updatedAt: Date;
+}
+
+export type CreateUserResponse = UserEntity;
```

Comment:
chyba wolalbym zebys zrobil osobny typ i tak nie bedziesz zwracal wszystkich danych jak password wiec przypisane jest chujowe tutaj

Reply:
Dokladnie jak Szymon pisze, typ musi sie zgadzac 1:1 z tym co zwracasz. Teraz tez tak patrze ze w entity brakuje Ci m.in. password, czy status. 😁